src/WebJobs.Extensions.DurableTask/DurableHttpRequest.cs

```diff
@@ -110,5 +111,68 @@ public override bool CanConvert(Type objectType)
                 writer.WriteValue(method.ToString());
             }
         }
+
+        private class TokenSourceConverter : JsonConverter
+        {
+            private enum TokenSourceType
+            {
+                None = 0,
```

Comment:
Is this just a placeholder, or do we ever see using this enum value?

Reply:
It's just a placeholder. I recently started a habit adding default enum values like this so that the absence of any value does not automatically get mapped to a real value.